imageroot/actions/configure-module/01_validate_domain

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+import os
+import sys
+import json
+import agent
+import urllib.request
+import urllib.error
+import ssl
+
+
+# retrieve json data
+data = json.load(sys.stdin)
+
+# Setup default values
+host = data.get("host")
+# do not test if it is the same host
+oldHost = os.environ.get('TRAEFIK_HOST','')
+if host == oldHost:
+    exit(0)
+
+
+# set error validation
+agent.set_weight(os.path.basename(__file__), 0)
+
+
+
+http = 0
+https = 0
+
+def testUrl(domain):
```

Comment:
Why using a function for this?
You just call it once.
Also I see no reason to use global vars.

Reply:
done